.circleci/config.yml

```diff
@@ -208,13 +232,13 @@ workflows:
             branches:
               ignore:
                 - master
-      - test-integration-windows:
-          name: Integration Tests Windows
-          context: nodejs-install
-          filters:
-            branches:
-              ignore:
-                - master
+#- test-integration-windows:
```

Comment:
Maybe adding comment on why it's commented:? :)

Reply:
It is in commit message, but sure :-)